routes/registration.js

```diff
@@ -15,61 +15,105 @@ const validateEmail = utilityFunctions.validateEmail;
 const validateName = utilityFunctions.validateName;
 const validatePassword = utilityFunctions.validatePassword;
 const randomStr = utilityFunctions.randomString;
+const readHTMLFile = utilityFunctions.readHTMLFile;
+const fs = require('fs');
+
+const codes = JSON.parse(fs.readFileSync('./data/codes.json'));
+const errorCode = codes.errors;
+const successCode = codes.success;
+
+const otherData = JSON.parse(fs.readFileSync('./data/english.json'));
+const mailData = otherData.mail;
+const mailFooterData = otherData.mail.footer;
+const mailHeaderData = otherData.mail.header;
 
 function verificationMail(req, res, rString) {
-    mail.mailOptions.to = req.body.email;
-    mail.mailOptions.subject = 'PC PrepKit Email Verification';
-    mail.mailOptions.html = `<b>Click on the link to complete the verification</b> <a href='${config.basePath}verification?token=${rString}&user=${req.body.email}'>Verify</a>`;
-    mail.smtpTransport.sendMail(mail.mailOptions, function(error) {
-        if (error) {
-            res.status(500).json({error: 'Something Went Wrong! Try again later.'});
-        } else {
-            res.json('Verification Mail Sent, Please check your mail.');
-        }
+    readHTMLFile('./public/pages/registration-verification.html', function(err, html) {
+        const template = handlebars.compile(html);
+        const replacements = {
+            host: req.headers.host,
+            token: rString,
+            email: req.body.email,
+            title: mailData.registration.title,
+            preHeader: mailData.registration.preHeader,
+            brandTitle1: mailHeaderData.brandTitle1,
+            brandTitle2: mailHeaderData.brandTitle2,
+            greeting: mailData.registration.greeting,
+            textContent1: mailData.registration.textContent.textContent1,
+            textContent2: mailData.registration.textContent.textContent2,
+            textContent3: mailData.registration.textContent.textContent3,
+            textContent4: mailData.registration.textContent.textContent4,
+            callToAction: mailData.registration.callToAction,
+            signatureGreet: mailData.registration.signature.endGreet,
+            signatureSignOff: mailData.registration.signature.signOff,
+            footerText1: mailFooterData.textContent1,
+            footerText2: mailFooterData.copyrights
+        };
+
+        const htmlToSend = template(replacements);
+        mail.mailOptions.to = req.body.email;
+        mail.mailOptions.subject = 'PC PrepKit Email Verification';
```

Comment:
Still there are hard coded strings in the code

Reply:
Yes @visitha, Rajat had changed it in his Code Clean Up PR, So I did not update it as it would cause merge conflicts. 
You can see all these strings present in this file from his PR #45 
https://github.com/systers/PC-Prep-Kit/pull/45/files#diff-3eef044ccd5155e29baf18d4a7e46386